packages/web-pkg/src/components/CreateShortcutModal.vue

```diff
@@ -210,11 +182,11 @@ export default defineComponent({
       } catch (_) {}
     }
 
-    const dropItemResourceClicked = (resource: Resource) => {
+    const dropItemResourceClicked = ({ data }) => {
```

Comment:
Can you type the arg here, what does `data` look like?

Reply:
```suggestion
    const dropItemResourceClicked = ({ data }: SearchResultValue) => {
```